pkg/client/src/app/pages/applications/analysis-wizard/analysis-wizard.tsx

```diff
@@ -80,7 +84,6 @@ export const AnalysisWizard: React.FunctionComponent<IAnalysisWizard> = ({
 
   const [isInitTasks, setInitTasks] = React.useState(false);
   const [createdTasks, setCreatedTasks] = React.useState<Array<Task>>([]);
-  const isMutating = useIsMutating();
```

Comment:
Need to integrate this into the upload binary step so that the next button is disabled when uploading a binary.

Reply:
That's a good point.